### PR TITLE
ADAPT-000: GraphQL Build errors (#39)

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

Fixes build errors caused by no content of type.

RE: https://www.gatsbyjs.com/docs/reference/graphql-data-layer/schema-customization/#fixing-field-types

### CODE REVIEW

**Comment by sherakama** [comment]
@yvonnetangsu 

The build errors are caused when no content of something with a graphql query exists and gatsby cannot infer the field types. This will patch up the alerts and the redirects so that if there is no content (published or at all) it will still complete.

File: gatsby-config.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ module.exports = {
         accessToken: process.env.GATSBY_STORYBLOK_ACCESS_TOKEN,
         homeSlug: "home",
         resolveRelations: storyblokRelations,
-        version: process.env.NODE_ENV === "production" ? "published" : "draft", // show only published on the front end site
+        version: "published",
```
Comment (sherakama) [inline gatsby-config.js:65]: When using a public api key and this logic, the build fails even with the graphql type fields added. I figured until we get into workflows and drafts we can just rely on the API key access to account for what we see.

Comment (yvonnetangsu) [reply gatsby-config.js:65]: Thank you for investigating this, @sherakama !

**Review by yvonnetangsu — APPROVED** [review]
Works well for me. Thank you!

**Comment by yvonnetangsu** [comment]
@sherakama Are the global alerts live on Online Giving? I'm thinking we probably need this fix over there too.

**Comment by sherakama** [comment]
> @sherakama Are the global alerts live on Online Giving? I'm thinking we probably need this fix over there too.

Yep, going to copy this over there too.

**Pull request merged**